src/main/webapp/js/components/jupyter/GeppettoJupyter.js

```diff
@@ -13,7 +13,71 @@ define(function (require, exports, module) {
 	var GEPPETTO = require('geppetto');
 
 	var $ = require('jquery');
+	
+	var PopupModel = jupyter_widgets.WidgetModel.extend({
```

Comment:
I don't think you need a PopupModel. Actually, I think you are not using it anywhere